### DISCUSSION TITLE
/DELAYLOAD GDAL(D) is nu niet effectief; voorts: selectief laden van GDAL drivers en alleen proj4 tbv Convert, ter componentisering van laad-proces

### DISCUSSION THREAD

**Discussion by jipclaassens** [ObjectVision/GeoDMS#65] [Ideas]
/DELAYLOAD GDAL(D) is nu niet effectief; voorts: selectief laden van GDAL drivers en alleen proj4 tbv Convert, ter componentisering van laad-proces

**Comment by eoudejans** [comment]
The selective loading of known GDAL drivers has been build in since commit https://github.com/ObjectVision/GeoDMS/commit/0f16d606a9451c614c42e9da223161c5b85bcdca. To be released from GeoDMS version 8.6.3 and up.
